src/UpdateLocalizedResources.targets

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="UpdateXlf" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="SaveXlfToResx"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="UpdateXlfFromResx"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="EmitLocalizedResources"/>
+
+  <!-- Emit the localized EmbeddedResource items before the common targets start using them -->
+  <PropertyGroup>
+    <PrepareResourceNamesDependsOn>
```

Comment:
I also had a philosophical debate with myself about this one. Can I do `BeforeTargets="$(PrepareResourceNamesDependsOn)"`? I really want these two targets (that inject and create resource items and files) to be the very first, since most likely all the other targets in that property will depend on them.

Also, I want to reference the property and not peek inside it to see what's the first target there, since that feels a bit like breaking encapsulation, and other projects could preppend other targets in that string, making my first choice not "the first one" anymore.